velox/common/base/BitUtil.cpp

```diff
@@ -90,6 +90,18 @@ void copyBitsBackward(
   }
 }
 
+void toString(const uint32_t* bits, int offset, int size, char* out) {
```

Comment:
We don't need these 2, just cast to `const uint64_t*`

Reply:
done, and pushed new changes